src/server/system_services/schemas/namespace_resource_schema.js

```diff
@@ -52,6 +51,19 @@ module.exports = {
                 }
             }
         },
+        nsfs_config: {
+            type: 'object',
+            required: ['fs_path'],
+            properties: {
+                fs_path: {
+                    type: 'string'
+                },
+                fs_backend: {
+                    type: 'string',
+                    enum: ['CEPH_FS', 'GPFS']
```

Comment:
why is this defined here again? this meanss you want the nsfs_config definition to move to common_api and used in both.

Reply:
moved it, thanks